packages/ui-components/style/icons.css

```
@@ -206,39 +206,39 @@
 }
 
 .jp-jupyter-icon-color[fill] {
-  fill: var(--jp-jupyter-icon-color);
+  fill: var(--jp-jupyter-icon-color, var(--jp-warn-color0));
 }
 
 .jp-notebook-icon-color[fill] {
-  fill: var(--jp-notebook-icon-color);
+  fill: var(--jp-notebook-icon-color, var(--jp-warn-color0));
 }
 
 .jp-json-icon-color[fill] {
-  fill: var(--jp-json-icon-color);
+  fill: var(--jp-json-icon-color, var(--jp-warn-color1));
 }
 
 .jp-console-icon-color[fill] {
-  fill: var(--jp-console-icon-color);
+  fill: var(--jp-console-icon-color, white);
 }
 
 .jp-console-icon-background-color[fill] {
-  fill: var(--jp-console-icon-background-color);
+  fill: var(--jp-console-icon-background-color, var(--jp-brand-color1));
 }
 
 .jp-terminal-icon-color[fill] {
-  fill: var(--jp-terminal-icon-color);
+  fill: var(--jp-terminal-icon-color, var(--jp-layout-color2));
 }
 
 .jp-terminal-icon-background-color[fill] {
-  fill: var(--jp-terminal-icon-background-color);
+  fill: var(--jp-terminal-icon-background-color, var(--jp-inverse-layout2));
 }
 
 .jp-text-editor-icon-color[fill] {
-  fill: var(--jp-text-editor-icon-color);
+  fill: var(--jp-text-editor-icon-color), var(--jp-inverse-layout3));
```

Looks like the number of parenthesis is unbalanced:

```suggestion
  fill: var(--jp-text-editor-icon-color, var(--jp-inverse-layout3));
```